### PR TITLE
Content19

### PR DESCRIPTION
**Pull request by mustafaozger**

### CODE REVIEW

**Comment by dogusmiuw** [comment]
exclude feature 9 from this pull request. and send it again

**Comment by mustafaozger** [comment]
I've revert the feature9 commit

File: editor-components/content/content19/content19.tsx [hunk 1]
```diff
@@ -2,64 +2,205 @@ import * as React from "react";
...
+                          className: this.decorateCSS("play-image")
+                        }}></ComposerIcon>
+                      </div>
+                    )}
+                  </span>
+                </div>
+              </div>
+            )}
+            <div className={this.decorateCSS("text-side")}>
+              {this.getProp("title-text").value && (
```
Comment (dogusmiuw) [inline editor-components/content/content19/content19.tsx:156]:
```suggestion
              {this.getProp("title-text", { as_string: true }) && (
```

File: editor-components/content/content19/content19.tsx [hunk 2]
```diff
@@ -2,64 +2,205 @@ import * as React from "react";
...
+              <div className={this.decorateCSS("description-author")}>
+                {authorImage && (
+                  <div className={this.decorateCSS("author-icon")}>
+                    <ComposerIcon name={this.getPropValue("author-icon")} propsIcon={{
+                      className: this.decorateCSS("author-icon-photo")
+                    }}></ComposerIcon>
+                  </div>
+                )}
+                <div className={this.decorateCSS("author-info")}>
+                  <p className={"author-description-text"}>{this.getPropValue("author-description")}</p>
```
Comment (dogusmiuw) [inline editor-components/content/content19/content19.tsx:173]: don't render this if author-description is empty

File: editor-components/content/content19/content19.tsx [hunk 3]
```diff
@@ -2,64 +2,205 @@ import * as React from "react";
...
+                  <h1 className={"author--text"}>{this.getPropValue("author-name")}</h1>
```
Comment (dogusmiuw) [inline editor-components/content/content19/content19.tsx:174]: don't render this if author-name is empty

File: editor-components/content/content19/content19.tsx [hunk 4]
```diff
@@ -2,64 +2,205 @@ import * as React from "react";
...
+                  const buttonText = buttonData.find((item: any) => item.key === "buttonText")?.value || "";
+                  const url = buttonData.find((item: any) => item.key === "url")?.value || "";
+                  if (buttonText.trim() !== "") {
+                    return (
+                      <ComposerLink key={index} path={url}>
+                        <Button className={this.decorateCSS("button")}>{buttonText}</Button>
+                      </ComposerLink>
+                    );
+                  }
+                  return null;
```
Comment (dogusmiuw) [inline editor-components/content/content19/content19.tsx:190]:
```suggestion
```

File: editor-components/content/content19/content19.tsx [hunk 5]
```diff
@@ -2,64 +2,205 @@ import * as React from "react";
...
+                  </div>
+                )}
+                <div className={this.decorateCSS("author-info")}>
+                  <p className={"author-description-text"}>{this.getPropValue("author-description")}</p>
+                  <h1 className={"author--text"}>{this.getPropValue("author-name")}</h1>
+                </div>
+              </div>
+          
+              <div className={this.decorateCSS("button-container")}>
+                {(this.getProp("buttons").value as any[]).map((buttonObj, index) => {
```
Comment (dogusmiuw) [inline editor-components/content/content19/content19.tsx:179]: don't map arrays like this.
[check here](https://github.com/Teknodev/composer-tools/blob/main/editor-components/README.md#how-to-map-things)

File: editor-components/content/content19/content19.tsx [hunk 6]
```diff
@@ -2,64 +2,205 @@ import * as React from "react";
...
+                  const buttonData = buttonObj.value;
```
Comment (dogusmiuw) [inline editor-components/content/content19/content19.tsx:180]: don't access values like this -> asdf.value

**Review by dogusmiuw — CHANGES_REQUESTED** [review]
please handle changes

**Comment by mustafaozger** [comment]
I changed my code based on your feedback

File: editor-components/content/content19/content19.tsx [hunk 7]
```diff
@@ -2,64 +2,229 @@ import * as React from "react";
...
-            className={this.decorateCSS("iframe")}
-             allowFullScreen></iframe>
-              <h3 className={this.decorateCSS("below-title")}>{this.getPropValue("below-title")}</h3>
-              <h2 className={this.decorateCSS("below-text")}>{this.getPropValue("below-text")}</h2>
-            </div>
+            {videoImage && (
+              <div className={this.decorateCSS("video-part")}>
+                <img alt="video image" src={videoImage} className={this.decorateCSS("video-images")} />
+                <div className={this.decorateCSS("play-part")}>
+                  <span className="as-play" onClick={() => {
```
Comment (Ayazelar) [inline editor-components/content/content19/content19.tsx:148]: should use decorateCSS

File: editor-components/content/content19/content19.tsx [hunk 8]
```diff
@@ -2,64 +2,229 @@ import * as React from "react";
...
+                          className: this.decorateCSS("play-image")
+                        }}></ComposerIcon>
+                      </div>
+                    )}
+                  </span>
+                </div>
+              </div>
+            )}
+            {
+              (this.castToString(titleText) || this.castToString(description) || this.castToString(authorDescription) || this.castToString(authorName) || this.castToObject<ButtonProps[]>("buttons").length > 0) && (
```
Comment (Ayazelar) [inline editor-components/content/content19/content19.tsx:165]: use a variable name like "displaycontent" maybe

**Review by Ayazelar — CHANGES_REQUESTED** [review]
.

**Review by Ayazelar — APPROVED** [review]

**Review by dogusmiuw — APPROVED** [review]

**Pull request merged**